build/ios/scripts/build_arch.sh

```diff
@@ -3,48 +3,41 @@ set -e
 
 if [ $# -ne 2 ]; then
     echo "$0 platform arch" 1>&2
-    echo "Example: $0 iPhoneSimulator i386"
+    echo "Example: $0 iphonesimulator i386"
     exit 1
 fi
 
 PLATFORM=$1
 ARCH=$2
 
-if [ "$PLATFORM" = "iPhoneOS" ]; then
-    EXTRA_CONFIG="--host=arm-apple-darwin14 --target=arm-apple-darwin14"
+MINIOSVERSION="7.1"
+if [ "$PLATFORM" = "iphoneos" ]; then
+    EXTRA_CONFIG="--host=arm-apple-darwin14 --target=arm-apple-darwin14 --enable-static --disable-shared"
+    MINVERSION="-miphoneos-version-min=$MINIOSVERSION"
 elif [ "$ARCH" = "i386" ]; then
-    EXTRA_CONFIG="--host=i386-apple-darwin14 --target=i386-apple-darwin14"
+    EXTRA_CONFIG="--host=i386-apple-darwin14 --target=i386-apple-darwin14 --enable-static --disable-shared"
+    MINVERSION="-mios-simulator-version-min=$MINIOSVERSION"
 elif [ "$ARCH" = "x86_64" ]; then
-    EXTRA_CONFIG="--host=x86_64-apple-darwin14 --target=x86_64-apple-darwin14"
+    EXTRA_CONFIG="--host=x86_64-apple-darwin14 --target=x86_64-apple-darwin14 --enable-static --disable-shared"
```

Comment:
AFAIK `./configure` generated by autoconf enables both shared and static. So, what is the purpose of explicit `--enable-static`? Is this needed for specific libraries to build? Otherwise, I'll just `--disable-shared`.